packages/demo/src/components/examples/DatePickerExamples.tsx

```diff
@@ -32,6 +35,12 @@ const CalendarComponent: React.FunctionComponent = () => (
     </Section>
 );
 
+const CountdownComponent: React.FunctionComponent = () => (
+    <Section title="Countdown component" mods={['mod-form-top-bottom-padding']}>
+        <Calendar id="countdown-calendar" countdown calendarSelection={CALENDAR_COUNTDOWN_DATES} />
```

Comment:
Any thoughts on creating a `Countdown` component that reuses the `Calendar` under the hood and passes down all the required information? I think it would make a friendlier component.

I imagine a usage like this, free of any props
```tsx
<MonthlyCountdown />
```
I don't see why the props would change since in this use-case we'll always want to display the number of days until the end of the month.

What do you think?

Reply:
+1 for this

Reply:
I think it could be good since it could happens that we need to have a countdown for X days that span on more than one month at some point :thinking: 

Like for the trial period or for the campaigns in QPL :eye:  :thinking:

Reply:
Yeah I can do this - my original try was to have a countdown prop that was either `boolean` or a config object where you could pass in the end date but it was getting messy. And I like your suggestion @aiheon about use cases for X days, but that would need to go back to UX because in the case of the days remaining being more than just the current month, how would you display it? side by side / above and below / go back and forth etc. I can do what Gael suggested and move it to another component, then any future changes should be easier to work in

Reply:
Yes we are in agreement on this :+1: